docs/apache-airflow/authoring-and-scheduling/deferring.rst

```diff
@@ -97,22 +100,22 @@ Writing a deferrable operator takes a bit more work. There are some main points
 Triggering Deferral
 ~~~~~~~~~~~~~~~~~~~
 
-If you want to trigger deferral, at any place in your Operator you can call ``self.defer(trigger, method_name, kwargs, timeout)``, which will raise a special exception that Airflow will catch. The arguments are:
+If you want to trigger deferral, at any place in your operator, you can call ``self.defer(trigger, method_name, kwargs, timeout)``. This raises a special exception for Airflow. The arguments are:
 
-* ``trigger``: An instance of a Trigger that you wish to defer on. It will be serialized into the database.
-* ``method_name``: The method name on your Operator you want Airflow to call when it resumes.
-* ``kwargs``: Additional keyword arguments to pass to the method when it is called. Optional, defaults to ``{}``.
-* ``timeout``: A timedelta that specifies a timeout after which this deferral will fail, and fail the task instance. Optional, defaults to ``None``, meaning no timeout.
+* ``trigger``: An instance of a trigger that you want to defer. It will be serialized into the database.
```

Comment:
```suggestion
* ``trigger``: An instance of a trigger that you want to defer to. It will be serialized into the database.
```

We are deferring the operator, not the trigger. The trigger is the responsible party now. So maybe this?